modules/EsetTools.py

```
@@ -241,7 +256,7 @@ def confirmAccount(self):
                 token = parseToken(self.email_obj, self.driver, True, max_iter=100, delay=3)
                 self.driver.switch_to.window(self.window_handle)
             else:
-                token = parseToken(self.email_obj, eset_business=True, max_iter=100, delay=3) # 1secmail, developermail
+                token = parseToken(self.email_obj, eset_business=True, max_iter=100, delay=3) # 1secmail
```

Ruff

`parseToken` may be undefined, or defined from star imports

(F405)